--- Makefile	2020-10-26 14:47:53.261593618 +0800
+++ ./Makefile	2020-10-28 08:07:32.670928180 +0800
@@ -59,5 +59,8 @@
 	sed -i 's/\r*$$/\r/' accelerated-domains.china.adguardhome.conf google.china.adguardhome.conf apple.china.adguardhome.conf
 endif
 
+overture: raw
+	sed -e 's|\(.*\)|domain:\1|' accelerated-domains.china.raw.txt google.china.raw.txt apple.china.raw.txt > overture-rules.txt
+
 clean:
 	rm -f {accelerated-domains,google,apple}.china.*.conf {accelerated-domains,google,apple}.china.raw.txt dnscrypt-proxy-forwarding-rules.txt
